endorsement/dao/shared_drive.py

```diff
@@ -302,6 +302,19 @@ def reconcile_drive_quota(
     )
 
     if quota_actual != quota_correct:
+        # temporary check for prior to ITBill subscription deadline
+        in_grace_period = (
+            dt.datetime.now() < shared_drive_subscription_deadline()
+        )
+        has_no_subscription = get_or_load_active_subscription(sdr) is None
```

Comment:
This check considers any drive without a DEPLOYED ITBillSubscription to have no subscription.

I think this is what is desired...